app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        raise ValueError("Invalid command format. "
+                         "Please provide the command in the format:"
+                         " cp <source_file> <destination_file>")
+
+    _, source_file, test_file = parts
+
+    if source_file == test_file:
```

Comment:
Perform the `cp` command check here, do not use indexes for retrieving values on line 4